packages/skeleton/src/lib/utilities/Modal/Modal.svelte

```diff
@@ -218,11 +218,10 @@
 {#if $modalStore.length > 0}
 	{#key $modalStore}
 		<!-- Backdrop -->
-		<!-- FIXME: resolve a11y warnings -->
-		<!-- svelte-ignore a11y-no-static-element-interactions -->
 		<div
 			class="modal-backdrop {classesBackdrop}"
 			data-testid="modal-backdrop"
+			role="presentation"
```

Comment:
I think it's safe to set it like this since you can't use keyboard navigation to access backdrop - it's only for mk+b users and so it can be safely ignored by screen readers.

Reply:
I'd be cautious about adding a11y features in this PR, that's likely out of scope. Though I do agree we should likely include a role here. If we do, we probably need a means to set it globally and override per instance. Maybe create a new ticket for this?

A11y attributes can be tricky - we don't often add them without proper research. Essentially no attributes are typically better than wrong attributes when it comes to accessibility.